rs/ethereum/cketh/minter/src/state.rs

```diff
@@ -557,6 +557,18 @@ impl State {
     pub fn eth_balance(&self) -> &EthBalance {
         &self.eth_balance
     }
+
+    pub fn max_block_spread_for_logs_scraping(&self) -> u16 {
+        if self.evm_rpc_id.is_some() {
+            // Limit set by the EVM-RPC canister itself, see
+            // https://github.com/internet-computer-protocol/evm-rpc-canister/blob/3cce151d4c1338d83e6741afa354ccf11dff41e8/src/candid_rpc.rs#L192
+            500_u16
```

Comment:
Would it make sense to (be able to) query the EVM RPC canister for this value instead?

Reply:
It would help, but I think the best would be that the EVM RPC canister does the "paging" for you, e.g., the caller asks for logs between `block_1` and `block_3`, and the EVM RPC canister returns the logs between `block_1` and `block_2` and it's up to the caller to call the EVM RPC canister to fetch the logs in the interval `[block_2, block_3]`

Reply:
I agree! Obviously, that's something that can be addressed in future work.